test/api/unit/serializers/site.test.js

```diff
@@ -1,8 +1,9 @@
-const { expect } = require('chai');
+const { expect, assert } = require('chai');
```

Comment:
was this an automagically added `assert` ? Doesn't look like it is used.

Reply:
Definitely automagic.